rundmcmc/make_graph.py

```diff
@@ -136,14 +137,21 @@ def construct_graph_from_df(df,
     pops = 0
     if pop_col:
         pops = df[pop_col].to_dict()
+    else:
+        warnings.warn("No population column was given, assuming all 0")
 
-    areas = df['geometry'].area.to_dict()
     if area_col:
         areas = df[area_col].to_dict()
+    else:
+        # This may be slightly expensive, so only do it if we know we have to.
+        areas = df['geometry'].area.to_dict()
```

Comment:
fyi, geopandas generally recommends using `geodataframe.geometry` to refer to the active geometry column, since that is [always the active geometry column](https://github.com/geopandas/geopandas/issues/755).